pandas/tests/io/test_stata.py

```diff
@@ -2058,6 +2058,20 @@ def test_backward_compat_nodateconversion(version, datapath):
     tm.assert_frame_equal(old_dta, expected, check_dtype=False)
 
 
+@pytest.mark.parametrize("version", [102])
+def test_backward_compat_nostring(version, datapath):
+    # The Stata data format prior to 105 did not support a date format
+    # so read the raw values for comparison
+    data_base = datapath("io", "data", "stata")
+    ref = os.path.join(data_base, "stata-compat-118.dta")
+    old = os.path.join(data_base, f"stata-compat-{version}.dta")
```

Comment:
Could you make these 2 `datapath` as well?

Reply:
I have now made this change. Note that the equivalent tests for other format versions don't use ```datapath``` for these either, but I haven't changed them to keep this pull restricted to just 102 format related changes.